### PR TITLE
feat: Add About

### PR DESCRIPTION
**Pull request by yoobi**
Adding about activity for #25

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,14 +2,15 @@ plugins {
...
+    id "com.mikepenz.aboutlibraries.plugin"
 }
 
 android {
     compileSdk 33
 
     defaultConfig {
         applicationId "com.github.muellerma.mute_reminder"
-        minSdk 14
+        minSdk 21
```
Comment (mueller-ma) [inline app/build.gradle:13]: Which library requires this?

File: app/build.gradle [hunk 2]
```diff
@@ -48,6 +49,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.9.0'
-    implementation 'androidx.appcompat:appcompat:1.5.1'
```
Comment (mueller-ma) [inline app/build.gradle:52]: I'd keep appcompat here, so we have control about the version.

File: app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt [hunk 3]
```diff
@@ -14,8 +17,30 @@ class MainActivity : AppCompatActivity() {
             "https://github.com/mueller-ma/MuteReminder".openInBrowser(this)
         }
 
-        binding.version.text = getString(R.string.version, BuildConfig.VERSION_NAME)
+        binding.version.text = getString(R.string.app_version, BuildConfig.VERSION_NAME)
```
Comment (mueller-ma) [inline app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt:20]: This text can be removed now.

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -8,7 +8,7 @@
     <string name="main_activity_hint">Eine Benachrichtigung wird angezeigt, wenn der Klingelton stummgeschalten ist und Medien über lokale Lautsprecher abgespielt werden würden.</string>
     <string name="report_issues_github">Fehler auf GitHub melden</string>
     <string name="error_no_browser_found">Kein Browser gefunden</string>
-    <string name="version">Version %s</string>
+    <string name="app_version">Version %s</string>
```
Comment (mueller-ma) [inline app/src/main/res/values-de/strings.xml:11]: Can you revert the changes in `values-*/strings.xml`? These files are managed by Crowdin.

**Review by mueller-ma — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (yoobi) [reply app/build.gradle:13]: this one `implementation "com.mikepenz:aboutlibraries:$about_libraries_version"`

File: app/build.gradle (hunk 2 above)
Comment (yoobi) [reply app/build.gradle:52]: Keep appcompat comes with a bug where you need to declare 'androidx.lifecycle..' https://stackoverflow.com/questions/73406969/duplicate-class-androidx-lifecycle-viewmodellazy-found-in-modules-lifecycle-view

File: app/src/main/res/values-de/strings.xml (hunk 4 above)
Comment (yoobi) [reply app/src/main/res/values-de/strings.xml:11]: I'll remove the string as it is not needed anymore on the app

File: build.gradle [hunk 5]
```diff
@@ -1,14 +1,18 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext.kotlin_version = "1.7.20"
+    ext.about_libraries_version="10.4.0"
```
Comment (mueller-ma) [inline build.gradle:4]:
```suggestion
    ext.about_libraries_version = "10.4.0"
```

**Review by mueller-ma — APPROVED** [review]

File: app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt [hunk 6]
```diff
@@ -14,8 +17,28 @@ class MainActivity : AppCompatActivity() {
             "https://github.com/mueller-ma/MuteReminder".openInBrowser(this)
         }
 
-        binding.version.text = getString(R.string.version, BuildConfig.VERSION_NAME)
```
Comment (mueller-ma) [inline app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt:17]: This `TextView` should be removed from the xml as well.

Comment (yoobi) [reply app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt:17]: it has been removed yes

Comment (yoobi) [reply app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt:17]: Done
